Blazor/Components/PreFooter.razor

```diff
@@ -0,0 +1,12 @@
+﻿@inject IJSRuntime jsRuntime
+
+@code {
+    [Parameter]
+    public GoC.WebTemplate.Blazor.Model.PreFooter PreFooterObj { get; set; }
+
+    protected override void OnAfterRender(bool firstRender) // = On Page Load
+    {   
+        jsRuntime.InvokeVoidAsync("setPreFooter", JsonPlainSerializationHelper.SerializeToJson(PreFooterObj).ToString());
```

Comment:
This is more a question than a change request, but it applies to all components I've seen so far:   Could it cause a problem that we're not "awaiting" the async call here.    I mean that for everywhere we have InvokeAsync without await, but in particular here, where we then invoke "resetWetComponents" - I'm guessing it'd be a problem if setPreFooter is not done right?

Reply:
fixed